ckanext/iati/views/dashboard.py

```diff
@@ -159,16 +159,21 @@ def my_pending_organizations():
     try:
         if not c.user:
             raise NotAuthorized
-        pending_organizations = iati_h.organization_list_pending()
+
+        q = request.params.get(u'q', u'')
+        c.q = q
```

Comment:
Do you need this to be in the request context?  It's generally been deprecated, and you are passing this value into the render later as one of the extra vars.

Reply:
Actually, I'm not sure which one you're using, bout're referencing `c.` in the template, so the `extra_vars` might not be used at all.

Reply:
ok, so just remove 
`extra_vars["q"] = c.q`

Reply:
Maybe, I'm not sure if it's using the c. or the extra_vars version in the template.